app/Http/Controllers/GameController.php

```diff
@@ -9,16 +9,14 @@
 use App\Events\StartGameEvent;
 use App\Http\Requests\UpdateGameStateRequest;
 use App\Models\Game;
-use App\Models\User;
 use Carbon\Carbon;
+use OutOfBoundsException;
 
 class GameController extends Controller
 {
     public function getUsersInGame($gameId)
     {
-        return User::query()->whereHas('inviteKey', function ($query) use ($gameId) {
-            return $query->where('game_id', $gameId);
-        })->get();
+        return Game::find($gameId)->users();
```

Comment:
Hier zullen dadelijk wel merge-conflicts ontstaan, want ik heb dit ook aangepast in mijn PR. Maar waarschijnlijk is deze functie dan beter